library_panel.go

```diff
@@ -101,22 +101,30 @@ func (c *Client) LibraryPanels() ([]LibraryPanel, error) {
 
 // LibraryPanelByUID gets a library panel by UID.
 func (c *Client) LibraryPanelByUID(uid string) (*LibraryPanel, error) {
-	return c.panel(fmt.Sprintf("/api/library-elements/%s", uid))
+	resp := &LibraryPanelCreateResponse{}
+	path := fmt.Sprintf("/api/library-elements/%s", uid)
+
+	err := c.request("GET", path, nil, nil, &resp)
+	if err != nil {
+		return nil, err
+	}
+
+	return &resp.Result, err
 }
 
-// LibraryPanelByName gets a library panel by name.
+// LibraryPanelByName gets library panels by name.
```

Comment:
I wouldn't change the doc, the function itself is still return a single one

Reply:
0e2947e4e75a9c0fe664ab59d588bc745d09f744